model/src/main/kotlin/config/Excludes.kt

```diff
@@ -47,6 +47,14 @@ data class Excludes(
      */
     fun findProjectExclude(project: Project) = projects.find { it.path == project.definitionFilePath }
 
+    /**
+     * Return the [ScopeExclude]s for the provided [scope]. This includes global excludes as well as excludes in the
```

Comment:
Is 'global exclude' common terminology used in this code base?
...when reading just this sentence in isolation I was wondering what global actually means, e.g. looking into the implementation there isn't any mention of global in the code as well.

Reply:
Rephrased to make the meaning clearer.